schema/partner_category.js

```diff
@@ -24,12 +24,8 @@ const PartnerCategoryType = new GraphQLObjectType({
       type: GraphQLString,
     },
     partners: {
-      type: new GraphQLList(Partner.type),
-      args: {
-        size: {
-          type: GraphQLInt,
-        },
-      },
```

Comment:
Just have to remove the two unused types from this file to pass lint

```
/home/runner/metaphysics/schema/partner_category.js
   9:3  error  "GraphQLList" is defined but never used  no-unused-vars
  10:3  error  "GraphQLInt" is defined but never used   no-unused-vars
```

I also highly recommend installing SublimeLinter, which will highlight these problems as you're editing